package.json

```diff
@@ -30,6 +30,7 @@
   ],
   "devDependencies": {
     "@markedjs/html-differ": "^2.0.1",
+    "cheerio": "^1.0.0-rc.3",
```

Comment:
rc3?

Reply:
Should I change it to v0.22.0? either one works for what we need from it.

Reply:
It looks like v0.22.0 is 3 years old so lets keep it rc3